app/containers/App.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow
+ */
+
+import React, {Fragment, Component} from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  StatusBar,
+  Platform
+} from 'react-native';
+
+import {
+  Header,
+  LearnMoreLinks,
+  Colors,
+  DebugInstructions,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+import firebase from 'react-native-firebase';
+
+class App extends Component {
```

Comment:
All functions of firebase must be kept in a separate service file as a whole funtion. And that file should be imported in App.js. This code is clumsy.